packages/pricing/src/migrations/Migration20230928135653.ts

```diff
@@ -0,0 +1,37 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20230928135653 extends Migration {
+  async up(): Promise<void> {
+    this.addSql(
+      'alter table "price_rule" drop constraint "price_rule_price_set_id_foreign";'
+    )
+
+    this.addSql(
+      'alter table "price_set_money_amount" add column "number_rules" integer not null default 0;'
+    )
+
+    this.addSql(
+      'alter table "price_rule" drop constraint "price_rule_rule_type_id_unique";'
+    )
+    this.addSql(
+      'alter table "price_rule" add constraint "price_rule_price_set_id_foreign" foreign key ("price_set_id") references "price_set" ("id") on update cascade on delete cascade;'
+    )
+  }
+
+  async down(): Promise<void> {
+    this.addSql(
+      'alter table "price_rule" drop constraint "price_rule_price_set_id_foreign";'
```

Comment:
here is to add constraint again

Reply:
weird, somethings messed up in the migration generator. I'll creating one up from scratch.